### ISSUE TITLE
Wiki page to define a common language for screen reader specific modes and features

### ISSUE THREAD

**Issue by spectranaut** [w3c/aria-at#7]
Including things such as reading mode/iterative mode and their names in different screen readers.

**Comment by mfairchild365** [comment]
For whatever it's worth, I have a start to this.

* mappings of common commands across screen readers: 
  * [HTML representation](https://a11ysupport.io/learn/commands)
  * [JSON representation](https://a11ysupport.io/command_matrix.json)
* all of that is sourced from [ATBrowsers.json](https://github.com/accessibilitysupported/a11ysupport.io/blob/master/data/ATBrowsers.json)

Its a little messy and incomplete, but perhaps a start.

**Comment by spectranaut** [comment]
Matt started this here: https://github.com/w3c/aria-at/wiki/Screen-Reader-Terminology-Translation

**Comment by carmacleod** [comment]
I found [@ljwatson's post about screen reader interaction modes](https://tink.uk/understanding-screen-reader-interaction-modes/) helpful (it is very old, but still helps explain the concepts). :)

**Comment by zcorpan** [comment]
I'm not sure whether to have common terms for different ATs, or if it's better to use the terms that are specific for the AT in question. For example, we have tests for VoiceOver that say they're testing "interaction mode", but @cookiecrook pointed out (#111) that VO doesn't have modes, so it's a bit inaccurate to have common terms when the ATs have different idioms or ways of operation.
